### DISCUSSION TITLE
Meta Question - No template for templates?

### DISCUSSION THREAD

**Discussion by wekempf** [dotnet/templating#7143] [Template authoring]
I may just be missing something. After all, running `dotnet new search template` or even `dotnet new search templates` is far too generic and returns hundreds of false positives. Who knows, there may be an actual positive in that list, but I'm not seeing it. However, none of the documentation on authoring templates mentions using a template to start, so I assume if there is one buried in there, it's not an official template. This seems odd, to say the least. Project setup for template authoring isn't exactly trivial, so not having a template to quickly get you up and going seems... strange at best.

So, am I missing something, or is this really a gaping hole in all of this?

**Comment by sayedihashimi** [comment]
@wekempf you can use https://www.nuget.org/packages/Microsoft.TemplateEngine.Authoring.Templates/9.0.100-preview.3.24174.1. Some more info on usage at https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-template-package?pivots=dotnet-8-0
